packer-scripts/minimize

```diff
@@ -1,38 +1,47 @@
 #!/usr/bin/env bash
-
 set -o errexit
-set -o xtrace
 
-if [[ $PACKER_BUILDER_TYPE = googlecompute || $PACKER_BUILDER_TYPE = amazon-ebs ]]; then
-  echo "Skipping minimization on $PACKER_BUILDER_TYPE"
-  exit 0
-fi
+main() {
+  set -o xtrace
 
-if [[ $PACKER_BUILDER_TYPE != docker ]]; then
-  readonly SWAP_UUID=$(/sbin/blkid -o value -l -s UUID -t TYPE=swap)
+  if [[ "x${PACKER_BUILDER_TYPE}" = xgooglecompute || "x${PACKER_BUILDER_TYPE}" = xamazon-ebs ]]; then
```

Comment:
The `x${FOO}` step isn't really needed if you're using `[[` on bash, it's mostly an issue with `[` or `sh`. (see [Google's Shell Style Guide](https://google.github.io/styleguide/shell.xml#Testing_Strings)).